homeassistant/components/panel_iframe.py

```diff
@@ -8,22 +8,28 @@
 
 import voluptuous as vol
 
+from homeassistant.const import (CONF_ICON, CONF_URL)
 import homeassistant.helpers.config_validation as cv
 
-DOMAIN = 'panel_iframe'
 DEPENDENCIES = ['frontend']
 
+DOMAIN = 'panel_iframe'
+
 CONF_TITLE = 'title'
-CONF_ICON = 'icon'
-CONF_URL = 'url'
+
+CONF_ABSOLUTE_PATH_ERROR_MSG = "Invalid absolute path in relative URL"
```

Comment:
Message should be "Invalid relative URL."

Reply:
Thank you for your feedback. I used the term "absolute path in relative URL" because we are checking for an absolute path in a relative URL. I'll not discuss with you :-) Please check it again if this explanation makes sense to you, give "thumbs down" and I'll change.